__tests__/app.test.js

```
@@ -273,9 +273,41 @@ describe("PATCH /api/reviews/:review_id should change vote value", () => {
 			.send({ votes: 89 })
 			.expect(400)
 			.then(({ body }) => {
-				expect(body).toEqual({ msg: "Bad Request" });
+				expect(body).toEqual({ msg: "Bad request" });
 			});
 	});
 });
 
+describe("DELETE /api/comments/:comment_id", () => {
+	test("202 if deleting was a success ", () => {
+		return request(app)
```

This should return a 204 error